releasenotes/notes/bump-go-to-1.22.7-1002302b0ef8c246.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+- |
+    Agents are now built with Go ``1.22.7``.
```

Comment:
```suggestion
    Agents are now built with Go `1.22.7`.
```

Reply:
That seems like the correct formating for reStructuredText which `reno` is using

Reply:
indeed reno seems to prefer ``1.22.7``

Reply:
Sorry, my brain always goes MD first 😓